tests/test_dle.py

```diff
@@ -216,6 +217,26 @@ def test_cache_blocking_structure_optrelax_customdim():
                      'd,x0_blk0,y0_blk0,z0_blk0,x,y,z')
 
 
+def test_cache_blocking_structure_optrelax_defaultdim_alone():
```

Comment:
This is a horrible test name. How about `test_blocking_defaultdim_blockrelax`? Failing that, give it a shorter name and add a docstring explaining what it does.

Reply:
don't check the commit time because I had to rebase to address your other comment below. So you've to do with faith this time